bin/lisky

```diff
@@ -1,4 +1,24 @@
 #!/usr/bin/env node
```

Comment:
This should be removed from version control.

Reply:
One problem is that, permission of this file will be changed once we delete from version control.
When we `build` or `publish`, we should probably need to change the permission